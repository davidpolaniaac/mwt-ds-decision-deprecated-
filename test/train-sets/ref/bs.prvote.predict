0.478539 0.458390 1.000000
0.566167 0.452659 0.619416
0.406805 0.310507 0.456509
0.251622 0.173337 0.336450
0.322963 0.248373 0.432724
0.983781 0.935124 1.000000
0.336023 0.283262 0.428032
0.192092 0.045960 0.367021
0.412438 0.355603 0.473136
0.987273 0.949092 1.000000
0.299938 0.191055 0.379315
0.383079 0.324045 0.538324
0.282031 0.191435 0.419371
0.338635 0.310091 0.535169
0.944859 0.902809 0.994895
0.690543 0.592456 0.885632
0.944518 0.860310 1.000000
0.172964 0.057171 0.353277
0.329856 0.190818 0.644261
0.121756 0.045328 0.169037
0.262164 0.230530 0.923944
0.850630 0.611789 1.000000
0.193337 0.070341 0.310366
0.991279 0.489735 1.000000
0.063488 0.000000 0.151716
0.269885 0.138823 0.475542
0.281528 0.224301 0.449844
0.156322 0.044406 0.248419
0.854318 0.468885 0.996748
0.289275 0.243104 0.542972
0.864170 0.605581 1.000000
0.326018 0.138691 0.428626
0.164169 0.122979 0.198036
0.354337 0.272718 0.450172
0.851262 0.685431 0.997519
0.292653 0.175658 0.570998
0.843352 0.654157 1.000000
0.169977 0.099213 0.276944
0.895102 0.602038 1.000000
1.000000 1.000000 1.000000
0.257839 0.109665 0.408097
0.890388 0.826258 0.946926
0.052490 0.011032 0.125593
0.292537 0.147675 0.375115
0.169259 0.034709 0.309596
0.129500 0.083988 0.181821
0.257917 0.119488 0.701049
0.056381 0.043627 0.557042
0.937568 0.290648 0.961409
0.256776 0.222167 0.287373
0.959284 0.428921 1.000000
1.000000 0.368642 1.000000
0.127635 0.000000 0.401585
0.232054 0.092099 0.481334
0.992366 0.969464 1.000000
0.866602 0.222806 1.000000
0.193469 0.119060 0.317708
0.224850 0.069827 0.380972
0.412085 0.371972 0.885179
0.188716 0.067895 0.437237
0.887909 0.570298 1.000000
0.108563 0.016143 0.348595
0.714218 0.597393 0.885829
0.124156 0.066054 0.201452
0.905294 0.621175 1.000000
0.155813 0.093998 0.313378
0.331615 0.069193 0.460195
0.122725 0.095883 0.806661
0.114768 0.045854 0.196326
0.632674 0.566345 0.797911
0.158623 0.056481 0.425846
0.677401 0.539467 0.902148
0.325638 0.283821 0.976362
0.048461 0.000000 0.127378
0.890640 0.726537 1.000000
1.000000 0.459947 1.000000
0.164979 0.029138 0.371215
0.355033 0.187811 0.601652
0.100235 0.057713 0.119670
0.224388 0.059909 0.347789
0.048057 0.000000 0.192228
0.214221 0.141604 0.400031
0.913158 0.787474 1.000000
0.286814 0.102555 0.489686
0.173353 0.150986 0.222639
0.214987 0.117383 0.415650
0.364587 0.345910 1.000000
0.835789 0.664915 1.000000
0.943070 0.772280 1.000000
0.093061 0.000000 0.300062
0.142314 0.053952 0.209100
0.841398 0.534163 0.951390
1.000000 1.000000 1.000000
0.130773 0.072309 0.175301
0.798578 0.681810 1.000000
0.053998 0.000000 0.213567
0.768004 0.504172 1.000000
0.170715 0.027735 0.319510
0.921924 0.384518 0.950059
0.863378 0.533635 1.000000
0.132375 0.000000 0.401651
0.710231 0.534494 0.939360
0.040689 0.000000 0.104174
0.879732 0.493532 1.000000
0.083491 0.000000 0.147888
0.987010 0.963237 1.000000
0.056283 0.000000 0.143924
0.193988 0.048358 0.415193
0.166880 0.133686 0.228804
0.968765 0.917059 1.000000
0.837322 0.665323 1.000000
0.618216 0.154025 0.628948
0.106209 0.000000 0.408536
0.351225 0.317683 1.000000
0.056086 0.000000 0.509912
0.049482 0.000000 0.506964
0.886553 0.323823 0.987185
0.877494 0.452777 1.000000
0.403805 0.327837 1.000000
0.160269 0.001862 0.496232
0.119045 0.018962 0.388772
0.937201 0.927328 0.949857
0.125252 0.000000 0.404609
0.952529 0.453135 1.000000
0.699265 0.495457 0.991451
0.801212 0.369999 0.885172
0.133208 0.000000 0.405252
0.814467 0.449116 0.886408
0.158933 0.000000 0.400786
0.804983 0.414330 0.945832
0.043934 0.000000 0.117271
0.891290 0.733348 1.000000
0.137828 0.037800 0.243426
0.261271 0.237421 0.939561
0.103741 0.000000 0.367407
0.211073 0.026962 0.300478
0.855485 0.496068 1.000000
0.403767 0.350595 0.995408
0.987643 0.954148 1.000000
0.251342 0.029820 0.379896
0.218393 0.002710 0.456161
0.074298 0.000000 0.174533
0.791924 0.683823 0.909649
0.910751 0.823318 0.990685
0.944804 0.408713 0.985697
0.811008 0.611208 0.920679
0.921439 0.488054 0.945687
0.965981 0.892140 1.000000
0.080854 0.000000 0.323415
0.895920 0.309824 0.909166
0.952046 0.908086 1.000000
0.223357 0.215659 0.913834
0.832780 0.622499 0.926225
0.091201 0.000000 0.364806
0.583699 0.000000 0.663357
0.813661 0.528311 0.920641
0.974728 0.937749 1.000000
0.016536 0.000000 0.061637
0.962409 0.849636 1.000000
0.004912 0.000000 0.621811
0.839594 0.548149 0.956069
0.053547 0.000000 0.203684
0.179329 0.040461 0.379248
0.773867 0.678339 0.936412
0.095439 0.000532 0.331340
0.922309 0.409029 0.923825
0.976775 0.933654 1.000000
0.040832 0.000000 0.086990
0.457036 0.417787 0.997241
0.920089 0.705726 1.000000
0.712788 0.584154 0.913484
0.249767 0.029493 0.334615
0.118174 0.000000 0.273973
0.380593 0.314658 0.826037
0.117311 0.045142 0.200166
0.254937 0.031250 0.597997
0.150506 0.025126 0.275946
0.886845 0.774050 0.971336
0.907234 0.677971 1.000000
0.835232 0.733083 0.959718
0.814622 0.448380 0.996360
0.132508 0.083232 0.174466
0.892350 0.425506 0.908774
0.066416 0.000000 0.248872
0.170242 0.068516 0.384858
0.022225 0.000000 0.513376
0.840112 0.627244 1.000000
0.033030 0.000000 0.558645
0.211386 0.036497 0.363344
0.811269 0.671142 0.994183
0.792029 0.661582 0.920168
0.247624 0.075627 0.462856
0.012373 0.000000 0.035419
0.160132 0.000000 0.297109
0.188652 0.063500 0.297037
0.993022 0.982524 1.000000
0.468484 0.450480 0.962684
0.007095 0.000000 0.019079
0.219083 0.018797 0.458421
0.771669 0.532585 1.000000
