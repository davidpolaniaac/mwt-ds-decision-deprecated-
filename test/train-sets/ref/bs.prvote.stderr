only testing
Num weight bits = 18
learning rate = 10
initial_t = 1
power_t = 0.5
predictions = bs.prvote.predict
using no cache
Reading datafile = train-sets/0001.dat
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
0.271921   0.271921            1         1.0   1.0000   0.4785       51
0.296233   0.320545            2         2.0   0.0000   0.5662      104
0.205318   0.114402            4         4.0   0.0000   0.2516      135
0.134456   0.063595            8         8.0   0.0000   0.1921      146
0.110978   0.087500           16        16.0   1.0000   0.6905       24
0.092083   0.073187           32        32.0   0.0000   0.3260       32
0.078698   0.065313           64        64.0   0.0000   0.1242       61
0.067919   0.057140          128       128.0   1.0000   0.8145      106

finished run
number of examples per pass = 200
passes used = 1
weighted example sum = 200
weighted label sum = 0
average loss = 0.0644692
best constant = -0.00502513
total feature number = 15482
